### PR TITLE
Adds Schedule A Transaction Selection

### PR DESCRIPTION
**Pull request by toddlees**
Issue #228 

@mjtravers I tried andi on the page and it alerted me to the use of aria-hidden.  Based on this [article](http://web-accessibility.carnegiemuseums.org/code/accordions/) I believe primeng is using aria-hidden appropriately and the alert can be ignored

### CODE REVIEW

File: front-end/src/app/transactions/transaction-type-picker/transaction-type-picker.component.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import { LabelList } from 'app/shared/utils/label.utils';
+
+@Component({
+  selector: 'app-transaction-type-picker',
+  templateUrl: './transaction-type-picker.component.html',
+  styleUrls: ['./transaction-type-picker.component.scss'],
+})
+export class TransactionTypePickerComponent implements OnInit {
+  scheduleATransactionTypeLabels: LabelList = ScheduleATransactionTypeLabels;
+  report: F3xSummary | undefined;
```
Comment (mjtravers) [inline front-end/src/app/transactions/transaction-type-picker/transaction-type-picker.component.ts]: I think we can change this F3xSummary type to the Report interface type. Is this component only for F3X transactions or will their be transactions for other form types down the road? If not, we can leave it as F3xSummary

**Review by mjtravers — CHANGES_REQUESTED** [review]
Per our discussion, please rename the component so it specifically references being an F3X-centric component and the 4 code smells. Thx

**Review by mjtravers — CHANGES_REQUESTED** [review]
I am seeing errors in the unit test output. They don't stop the testing but it should be sorted out before merging to develop. Here's a screen shot:
cr-app228-unit-test-errors

**Comment by toddlees** [comment]
image
@mjtravers addressed the messages coming up during the unit tests.  adding routertestingmodule silenced them

**Review by mjtravers — APPROVED** [review]

**Pull request merged**